scripts/fiber/tests-passing.txt

```diff
@@ -1122,26 +1115,11 @@ src/renderers/dom/shared/__tests__/ReactDOMServerIntegration-test.js
 * renders no empty styles with clean client render
 * renders no empty styles with client render on top of good server markup
 * renders no empty styles with client render on top of bad server markup
-* renders simple strings with server string render
-* renders simple strings with server stream render
-* renders simple strings with clean client render
-* renders simple strings with client render on top of good server markup
-* renders simple strings with client render on top of bad server markup
-* renders aria string prop with false value with server string render
-* renders aria string prop with false value with server stream render
-* renders aria string prop with false value with clean client render
-* renders aria string prop with false value with client render on top of good server markup
-* renders aria string prop with false value with client render on top of bad server markup
-* renders no aria prop with null value with server string render
-* renders no aria prop with null value with server stream render
-* renders no aria prop with null value with clean client render
-* renders no aria prop with null value with client render on top of good server markup
-* renders no aria prop with null value with client render on top of bad server markup
-* renders no unknown attributes with server string render
-* renders no unknown attributes with server stream render
-* renders no unknown attributes with clean client render
-* renders no unknown attributes with client render on top of good server markup
-* renders no unknown attributes with client render on top of bad server markup
+* renders unknown attributes with server string render
+* renders unknown attributes with server stream render
+* renders unknown attributes with clean client render
+* renders unknown attributes with client render on top of good server markup
+* renders unknown attributes with client render on top of bad server markup
 * renders unknown data- attributes with server string render
 * renders unknown data- attributes with server stream render
 * renders unknown data- attributes with clean client render
```

Comment:
This file is misleading. I've skipped some tests related to the unknown attribute list and ARIA rules. I think we could enable these again by moving the attributes we eliminate from the whitelist over to the attribute warning hooks.